src/objects/Hierarchy_Settings__c.object

```diff
@@ -436,6 +436,15 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>Prevent_Test_Deletion__c</fullName>
+        <defaultValue>false</defaultValue>
```

Comment:
I thought we wanted it to be enabled by default?

Reply:
My understanding from the spec doc is it should be disabled for new and existing customers: 
"Ensure that Prevent_Test_Deletion__c is disabled for new and existing customers."

I'm happy to update if it was changed.  Is the spec doc the source of truth?

Reply:
that makes sense.